container/container.go

```diff
@@ -188,6 +188,13 @@ func (container *Container) SetupWorkingDirectory() error {
 	if container.Config.WorkingDir == "" {
 		return nil
 	}
+
+	// If can't mount container FS at this point (eg Hyper-V Containers on
+	// Windows) bail out now with no action.
+	if !container.canMountContainerFS() {
```

Comment:
I would also usually not include `container` in something that is already part of `container`... so maybe `canMountRootFS` or `container.IsMountable`.